packaging/repoconfig/netdata-repo.spec

```diff
@@ -63,6 +65,11 @@ install -pm 644 %{SOURCE8} ./netdata.repo
 install -pm 644 %{SOURCE9} ./netdata-edge.repo
 %endif
 
+%if 0%{?amzn2023}
+install -pm 644 %{SOURCE10} ./netdata.repo
+install -pm 644 %{SOURCE11} ./netdata-edge.repo
+%endif
+
```

Comment:
While this currently works, it only does so because of the ordering. Given this, this should probably be nested within the generic Amazon Linux case above just to ensure that we’re not relying on overwriting files for it to look.

Reply:
Fair enough. Bugs me a bit as well.
In Amazon Linux 2 the macros `centos_ver`, `amzn2` and `amzn` are defined.
Amazon Linux 2023 defines `fedora`, `amzn2023` and `amzn`.

So the current master branch version already relies on overwriting files, i.e. the centos/fedora files are overwritten respectively.

Should I nest `amzn2` in `centos_ver` and `amzn2023` in `fedora`? Or do you prefer to distinguish all four different cases in this kind of style: `%if %[0%{?fedora} && !0%{?amzn2023}]`.

Reply:
Ah, yes, I forgot about that. In that case I think it’s probably better to update the existing Amazon Linux case to match on `amzn2` being defined, as that makes it clear what’s being checked for, and I’ll look into tidying up the other cases at some point in the future.